### DISCUSSION TITLE
unable to use search engine

### DISCUSSION THREAD

**Discussion by p060477** [qbittorrent/search-plugins#234] [Q&A]
hi, i've qbittorrent 4.3.4.1 -portable-
and i've also python on my pc but is also it no installed but -portable-
up to few days ago the search engine works...now no...and i'm not able to make the plug in .py working...i've python 3 on my pc..so what have i to do...??...consider that i really do not want to install or qbvittorent or python..i have them in portable way and alweays working up to few days ago...
thxs in adv
cheers

**Comment by elvisisvan** [comment]
do you have any visual demonstration of the output .py not working?

**Comment by servimo** [comment]
I am getting this, when trying to update plugins:

"Sorry, couldn't check for plugin updates. The update server is temporarily unavailable. The remote content was not found at the server (404)"

**Comment by hoot78** [comment]
Getting the same error on both Windows and TrueNAS Scale clients for the past 3-4 days.

**Comment by servimo** [comment]
Today the search is working.

**Comment by hoot78** [comment]
It's not the search that is broken, it's the search plugin update.

![image](https://user-images.githubusercontent.com/131098258/232930733-d8df2d28-410c-4397-86b3-64dc64e9176d.png)
![image](https://user-images.githubusercontent.com/131098258/232930804-3d1a0326-6c93-4c7f-b5fb-4083f3d2884f.png)

**Comment by hoot78** [comment]
Issue is resolved as of today.
